Paladin/TemplateWindow.cpp

```diff
@@ -71,6 +71,11 @@ TemplateWindow::TemplateWindow(const BRect& frame)
 		templatePath<<"Templates";
 		fTempList.ScanFolder(templatePath.GetFullPath());
 	}
+	
+	// #312 allow user specified templates
+	fTempList.ScanFolder("/boot/home/config/settings/Paladin/Templates");
```

Comment:
You shouldn't hard code paths, but use something like:

```
BPath path;
if (find_directory(B_USER_SETTINGS_DIRECTORY, &path) == B_OK) {
}
```

There's also BPathFinder to go through all possible locations of a file, see e.g.
https://github.com/HaikuArchives/BurnItNow/blob/master/Source/BurnWindow.cpp#L557